Lib/multiprocessing/semaphore_tracker.py

```diff
@@ -68,7 +70,11 @@ def ensure_running(self):
                 exe = spawn.get_executable()
                 args = [exe] + util._args_from_interpreter_flags()
                 args += ['-c', cmd % r]
+                if _HAVE_SIGMASK:
```

Comment:
A comment is needed to explain why you block signals. Add a reference to the bpo and explain the race condition.

Reply:
Done!